packages/compartment-mapper/src/parse-pre-cjs.js

```diff
@@ -35,19 +35,24 @@ export const parsePreCjs = async (
   const execute = async (moduleExports, compartment, resolvedImports) => {
     const functor = compartment.evaluate(source);
 
+    let exportsReferenceCopy = moduleExports;
```

Comment:
Did we not change these names to `originalExports` and `finalExports` in a different change?

Reply:
yes. I'm gonna need to rebase this, they were created in parallel